sdks/java/io/google-cloud-platform/pom.xml

```diff
@@ -82,11 +82,28 @@
     </dependency>
 
     <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-common-protos</artifactId>
+      <version>${grpc-google-common-protos.version}</version>
```

Comment:
no versions down here -- they should all be in top-level dependency management like you did for the `google-cloud-spanner` module.